base/floatfuncs.jl

```diff
@@ -234,10 +234,11 @@ the square root of [`eps`](@ref) of the type of `x` or `y`, whichever is bigger
 This corresponds to requiring equality of about half of the significand digits. Otherwise,
 e.g. for integer arguments or if an `atol > 0` is supplied, `rtol` defaults to zero.
 
+If `x` and `y` are scalar values (i.e., numbers), `norm` defaults to `abs`, but
```

Comment:
Maybe rephrase as:
```
The `norm` keyword defaults to `abs` for numeric `(x,y)` and to `LinearAlgebra.norm` for
arrays (where an alternative `norm` choice is sometimes useful). When `x` and `y` are arrays, …
```
to emphasize the fact that alternative norms are really only useful for arrays.

Reply:
Thank you for the suggestion. I originally wanted to minimize the changes to the existing documentation, but your version is much clearer & more useful. I have updated the PR accordingly.